yarn.lock

```diff
@@ -8998,13 +8993,6 @@ supports-hyperlinks@^2.0.0:
     has-flag "^4.0.0"
     supports-color "^7.0.0"
 
-swr@^1.0.1:
```

Comment:
I don't understand why we need swr. We can reverse this removal if there is an actual reason for us to use it.